_posts/2016-04-08-foreman-community-newsletter-april-2016.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: Foreman Community Newsletter - April 2016
+author: Amir Fefer
+tags:
+- foreman
+- newsletter
+excerpt: |
+  Foreman 1.11 and 1.10.3 RC2 released. Read on for details of new plugin releases and new recordings to watch!
```

Comment:
I don't believe we do RCs for z-stream releases :) What is 1.10.3rc2 meant to refer to?